openvmm/openvmm_entry/src/cli_args.rs

```diff
@@ -626,6 +626,17 @@ pub enum DiskCliKind {
     Memory(u64),
     // memdiff:<kind>
     MemoryDiff(Box<DiskCliKind>),
+    // sql:<path>[;create=<len>]
+    Sqlite {
+        path: PathBuf,
+        create_with_len: Option<u64>,
+    },
+    // sqldiff:<path>[;create]:<kind>
```

Comment:
We should probably standardize our parsing story a bit.

I'll note that qemu uses comma separated key-value params.

Reply:
Agreed, but I think that sort of thing should fall under the umbrella of a separate, CLI-focused PR.

I think its totally viable to use commas here, but it'll rely on the nested-parsers being a bit more clever (e.g: munching input incrementally, vs. having a top-level `split(',')` that restricts the use of commas in nested expressions).